starship

```diff
@@ -67,10 +69,12 @@ if __name__ == "__main__":
             input(PROMPTS["authentication_token"].format(config.api["authentication_token"]))
             or config.api["authentication_token"]
         )
-        save = input(PROMPTS["save"]).upper() == "Y"
+        port = input(PROMPTS["port"].format(config.api["port"])) or config.api["port"]
+        use_ssl = input(PROMPTS["use_ssl"].format("y" if config.api["use_ssl"] else "n")).upper() == "Y"
+        save = input(PROMPTS["save"].format("y" if config.api["use_ssl"] else "n")).upper() == "Y"
```

Comment:
@josh146 I guess this one's up to you and me now.
Any idea why we would consider `"use_sll"` when determining default prompt for  `save`?

Reply:
`use_ssl` is stored in the config file, and the preference remembered for next time.

On a tangent though, is this argument needed? Is there ever a use case where SF will not be connecting using SSL?

Reply:
🤷‍♂

Reply:
actually, I had issues using SSL, so I disabled it for my tests. But I believe forcing its use would just make the user debug the problem and eventually use it.

Reply:
The platform will force the use of SSL for security reasons, so in practice the only way you will be able to connect to platform is with SSL, unless you are developing against services locally. I agree that this option should be available for debugging purposes, but in production anyone not connecting over SSL will be refused.